src/dao/hooks/useDAOProposals.tsx

```diff
@@ -139,8 +139,7 @@ export function useDAOProposals(
             transactionType,
           } = proposalInfo;
           const { amount, receiver, token } = data ?? {};
-          const safeEVMAddress = await DexService.fetchContractEVMAddress(safeAccountId);
-          const threshold = await getThreshold(safeEVMAddress);
+          const threshold = await getThreshold(safeAccountId);
```

Comment:
Please rename the function parameter to account id form currently it is `getThreshold(safeAddress: string)`

Reply:
Done